org.eclipse.m2e.core.ui/src/org/eclipse/m2e/core/ui/internal/archetype/ArchetypeGeneratorImpl.java

```diff
@@ -64,19 +65,10 @@ public class ArchetypeGenerator {
   @Reference
   IMavenLauncher mavenLauncher;
 
-  /**
-   * Creates project structure using Archetype and then imports created project(s)
-   *
-   * @return a list of created projects.
-   * @since 1.8
+  /* (non-Javadoc)
+   * @see org.eclipse.m2e.core.ui.internal.archetype.ArchetypeGeneratorInterface#createArchetypeProjects(org.eclipse.core.runtime.IPath, org.eclipse.m2e.core.project.IArchetype, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.util.Properties, boolean, org.eclipse.core.runtime.IProgressMonitor)
    */
```

Comment:
This is not necessary and should be removed. @Override is enough here to reference documentation from parent type.

Reply:
Fixed in https://github.com/eclipse-m2e/m2e-core/pull/922/commits/4685cbd46790097615112e789598bf0315e1d3a8.